src/main/java/uk/gov/hmcts/befta/AuthenticationRetryConfiguration.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.befta;
+
+import org.apache.commons.lang3.math.NumberUtils;
+
+public class AuthenticationRetryConfiguration {
+    private int retryAttempts;
+    private int retryMaxTimeInSeconds;
+    private int retryMultiplierInMilliseconds;
+
+    public AuthenticationRetryConfiguration(String retryAttempts,
```

Comment:
Could you also update the readme file for these new configuration elements?

Reply:
Added  description in the readme